spec/build/git/clone_spec.rb

```diff
@@ -175,4 +175,20 @@
       it { store_example(name: 'git autocrlf invlaid') }
     end
   end
+
+  describe 'git.symlink option' do
+    context 'when it does not exist' do
+      it { should_not include_sexp [:cmd, /^git config --global core\.symlink/]}
+    end
+
+    context 'when it is set to true' do
+      before { payload[:config][:git]['symlink'] = true }
+      it { should include_sexp [:cmd, /^git config --global core\.symlinks true/]}
+    end
+
+    context 'when it is set to false' do
+      before { payload[:config][:git]['symlink'] = false }
+      it { should include_sexp [:cmd, /^git config --global core\.symlinks false/]}
```

Comment:
The Git option is [`core.symlinks`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-coresymlinks).
The config option is also called "`symlinks`"  is other PRs linked from https://travis-ci.community/t/git-symlinks-support/274/3.

(The 1st test is actuall funny in that it's always passes. It's a known fact in science that it's impossible to prove a negative :-) )

Reply:
(same goes for the other changed file)